Seymour/Matroid/Notions/Regularity.lean

```diff
@@ -75,7 +75,14 @@ private def Matrix.auxZ2_unexpand : Lean.PrettyPrinter.Unexpander
   | `($_ $x) => `($(x).$(Lean.mkIdent `auxZ2))
   | _ => throw ()
 
-variable {α : Type} [DecidableEq α]
+variable {α β : Type}
```

Comment:
Please make `β` local to the single declaration.

Reply:
Actually, I am doing a small refactor here, so I will push this change into your branch.